src/styles/_catalog.scss

```
@@ -20,3 +20,27 @@
     margin-right: 48px;
   }
 }
+
+@media (min-width: 1024px) {
+  .catalog {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+@media (max-width: 1023px) {
+  .catalog {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (max-width: 767px) {
+  .catalog {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (max-width: 487px) {
+  .catalog {
+    grid-template-columns: repeat(1, 200px);
+  }
```

The previous comment is not fixed. Use all the advantages of SASS, and don't duplicate yourself
Also, you can optimize your code by adding default styles for the mobile version and changing the number of columns when the screen width increases.
```suggestion
grid-template-columns: repeat(1, 200px);

@media (min-width: 486px) {
    grid-template-columns: repeat(2, 200px);
}

@media (min-width: 768px) {
    grid-template-columns: repeat(3, 200px);
 }

@media (min-width: 1024px) {
    grid-template-columns: repeat(4, 200px);
  }
}
```